public_log_parser.php

```diff
@@ -472,9 +472,8 @@ function fill_round_zip($file, $basename, &$roundzip, $day, $round) {
 							case 'cargo.html':
 							case 'gravity.html':
 							case 'records.html':
-							case 'singulo.html':
 							case 'experimentor.html':
-							case 'supermatter.html':
+							case 'engine.html':
```

Comment:
parsed logs has to be able to parse old olds, so you should just add engine.html and keep the old ones.